evaluation/src/main/java/nl/tudelft/serg/evosql/evaluation/tools/AluraTableXMLFormatter.java

```diff
@@ -0,0 +1,49 @@
+package nl.tudelft.serg.evosql.evaluation.tools;
+
+import nl.tudelft.serg.evosql.db.ISchemaExtractor;
+import nl.tudelft.serg.evosql.db.TableXMLFormatter;
+import nl.tudelft.serg.evosql.sql.ColumnSchema;
+import nl.tudelft.serg.evosql.sql.TableSchema;
+
+public class AluraTableXMLFormatter extends TableXMLFormatter {
+    private Crypto crypto;
+
+    public AluraTableXMLFormatter(ISchemaExtractor schemaExtractor, String query, Crypto crypto) {
+        super(schemaExtractor, query);
+        this.crypto = crypto;
+    }
+
+    @Override
+    protected String getTableSchemaXml(TableSchema tableSchema) {
```

Comment:
Should this method and the one below really be `protected`?

Reply:
I'm not sure what the original reasons were for making them `protected`. I suppose it could be of use if we would ever extend this functionality with subclasses, however it is currently unused. What would you suggest?

Reply:
It's mostly a consistency thing as the other class has public methods.

Reply:
Never mind, I'm bad at looking at access levels ;)